SkillChat.Server/NETVersionComparison.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Threading;
+
+namespace SkillChat.Server
+{
+    public class NETVersionComparison
+    {
+        /// <summary>
+        /// Получение наиболее подходящего Runtime .NET
+        /// </summary>
+        public static Runtime GetBestDotNETRuntime(string cfgVersion)
+        {
+            Runtime cfgRuntime = new Runtime() {Version = cfgVersion};
+            List<Runtime> _runtimesInstaled = new List<Runtime>();
+            List<Runtime> _NETCoreRuntimes = new List<Runtime>();
+
+            GetPackageList(_runtimesInstaled);
+
+            foreach (var item in _runtimesInstaled)
+            {
+                if (item.Name == "Microsoft.NETCore.App")
+                {
+                    _NETCoreRuntimes.Add(item);
+                }
+            }
```

Comment:
Вместо этой конструкции можно было использовать Linq метод для фильтрации - Where

```suggestion
            _NETCoreRuntimes = _runtimesInstaled.Where(runtime => runtime.Name == "Microsoft.NETCore.App").ToList();
```